### DISCUSSION TITLE
Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':                     No such file or directory @ dir_chdir - /github/workspace/docs

### DISCUSSION THREAD

**Discussion by chaitu786** [mmistakes/minimal-mistakes#3671] [General]
Iam getting this error Can you please help on this
Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':                     No such file or directory @ dir_chdir - /github/workspace/docs

**Comment by iBug** [comment]
Please provide a repository where we can look at. It's impossible to diagnose anything without reproducing it.

**Reply by chaitu786** [reply]
https://github.com/chaitu786/chaitu786.git  
Pls find my repository link

**Reply by iBug** [reply]
This repository looks nowhere like the theme. Are you sure you're using [Minimal Mistakes theme](https://mmistakes.github.io/minimal-mistakes/)? Otherwise you've been to the wrong place for your question.

**Reply by chaitu786** [reply]
no u can see its was giving one error ok jkyll only

**Reply by M1racle04** [reply]
were you able to resolve this error?
My repo is also showing the same error

**Reply by M1racle04** [reply]
Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
                    No such file or directory @ dir_chdir - /github/workspace/docs
/usr/local/bundle/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb:86:in `chdir': No such file or directory @ dir_chdir - /github/workspace/docs (Errno::ENOENT)

**Reply by M1racle04** [reply]
https://github.com/M1racle04/TinDog

this is the repo.
Please if you can look into it

**Reply by chaitu786** [reply]
just change the theme in github pages hosting them it will work
i think minimal theme is best to github page hosting

**Reply by chaitu786** [reply]
if you face still any errors You can reach-out to me
